pkg/core/native/native_neo.go

```diff
@@ -446,7 +450,7 @@ func (n *NEO) getNextBlockValidators(ic *interop.Context, _ []vm.StackItem) vm.S
 func (n *NEO) GetNextBlockValidatorsInternal(bc blockchainer.Blockchainer, d dao.DAO) (keys.PublicKeys, error) {
 	si := d.GetStorageItem(n.Hash, nextValidatorsKey)
 	if si == nil {
-		return bc.GetStandByValidators()
```

Comment:
If `validatorsCount` is not yet initialized, `GetValidatorsInternal` can fail, no?
https://github.com/neo-project/neo/blob/master/src/neo/SmartContract/Native/Tokens/NeoToken.cs#L264

Reply:
It's initialized now in `Initialize` at block 0, so I don't think we have any problem.